app/src/main/java/com/example/activitytimer/MainActivity.kt

```diff
@@ -26,9 +32,15 @@ class MainActivity : AppCompatActivity() {
 
         setSupportActionBar(binding.toolbar)
 
+        Log.d("ViewP", "main")
+        val adapter = ViewPagerAdapter(supportFragmentManager)
+        binding.pager.adapter = adapter
+        binding.tabLayout.setupWithViewPager(binding.pager)
+        Log.d("ViewP", "after ViewP")
+/*
```

Comment:
dont leave comments - delete these. you can get them from git history if you need them later (but you probably won't)